kolibri/plugins/learn/assets/src/views/index.vue

```diff
@@ -39,26 +39,28 @@
 
     <div v-if="!isSearchPage">
       <tabs>
-        <tab-link
-          type="icon-and-title"
-          :title="$tr('recommended')"
-          icon="forum"
-          :link="recommendedLink"
-        />
-        <tab-link
-          type="icon-and-title"
-          :title="$tr('topics')"
-          icon="folder"
-          :link="topicsLink"
-        />
-        <tab-link
-          name="exam-link"
-          v-if="isUserLoggedIn && userHasMemberships"
-          type="icon-and-title"
-          :title="$tr('exams')"
-          icon="assignments"
-          :link="examsLink"
-        />
+        <template scope="tab">
+          <component
+            :is="tab.link"
```

Comment:
I don't like how this change switches to using two advanced vue features - the `:is` attribute on `component` and the `scope` attribute on `template`. I think these might reduce readability more than what's gained by removing the explicit `tab-link` child component reference.